src/fswAlgorithms/orbitControl/lambertSecondDV/_UnitTest/test_lambertSecondDV.py

```diff
@@ -0,0 +1,162 @@
+    r = 10000. * 1000  # meters
+    if ecc < 1.0:
+        # elliptic case
+        oe1.a = r
+    else:
+        # parabolic and hyperbolic case
+        oe1.a = -r
+    oe1.e = ecc
+    oe1.i = 10. * macros.D2R
+    oe1.Omega = 20. * macros.D2R
+    oe1.omega = 30. * macros.D2R
+    oe1.f = 40. * macros.D2R
+    # spacecraft state at maneuver time, just before maneuver
+    r1_BN_N, v1_BN_N = orbitalMotion.elem2rv_parab(muBody, oe1)
+
+    # spacecraft state at maneuver time, just after maneuver
+    v2_BN_N = v1_BN_N + dv_N
+
+    # setup module to be tested
+    module = lambertSecondDV.LambertSecondDV()
+    module.ModelTag = "lambertSecondDV"
+    unitTestSim.AddModelToTask(unitTaskName, module)
+
+    # Configure input messages
+    lambertSolutionInMsgData = messaging.LambertSolutionMsgPayload()
+    lambertSolutionInMsgData.v2 = v1_BN_N
+    lambertSolutionInMsgData.valid = validLambert
+    lambertSolutionInMsg = messaging.LambertSolutionMsg().write(lambertSolutionInMsgData)
+
+    desiredVelocityInMsgData = messaging.DesiredVelocityMsgPayload()
+    desiredVelocityInMsgData.vDesired_N = v2_BN_N
+    desiredVelocityInMsgData.maneuverTime = tm
+    desiredVelocityInMsg = messaging.DesiredVelocityMsg().write(desiredVelocityInMsgData)
+
+    # subscribe input messages to module
+    module.lambertSolutionInMsg.subscribeTo(lambertSolutionInMsg)
+    module.desiredVelocityInMsg.subscribeTo(desiredVelocityInMsg)
+
+    # setup output message recorder objects
+    dvBurnCmdOutMsgRec = module.dvBurnCmdOutMsg.recorder()
+    unitTestSim.AddModelToTask(unitTaskName, dvBurnCmdOutMsgRec)
+
+    unitTestSim.InitializeSimulation()
+    unitTestSim.TotalSim.SingleStepProcesses()
+
+    # pull module data
+    dv = dvBurnCmdOutMsgRec.dvInrtlCmd[0]  # commanded Delta-V
+    burnStartTime = dvBurnCmdOutMsgRec.burnStartTime[0]
+
+    if validLambert == 1:
+        dvTrue = dv_N
+        burnStartTimeTrue = macros.sec2nano(tm)
+    else:
+        dvTrue = np.array([0., 0., 0.])
+        burnStartTimeTrue = 0
+
+    # make sure module output data is correct
+    paramsString = ' for DV={}, maneuver time={}, valid flag={}, accuracy={}'.format(
+        str(p1_dv),
+        str(p2_tm),
+        str(p3_valid),
+        str(accuracy))
+
+    np.testing.assert_allclose(dv,
+                               dvTrue,
+                               rtol=0,
+                               atol=accuracy,
+                               err_msg=('Variable: dv,' + paramsString),
+                               verbose=True)
+
+    np.testing.assert_allclose(burnStartTime,
+                               burnStartTimeTrue,
+                               rtol=0,
+                               atol=accuracy,
+                               err_msg=('Variable: burnStartTime,' + paramsString),
+                               verbose=True)
+
+
+if __name__ == "__main__":
+    test_lambertSecondDV(False, DVs[1], time_maneuver[0], 1, 1e-4)
```

Comment:
Extra line?

Reply:
I've noticed before that Github doesn't show the extra line at the end of the file? There is one. If you refer to the two empty lines in lines 159-160, Clion and PyCharm recommend using two empty lines after a class or function definition (which is the case here). Will mark as resolved.